src/utils/jobs.js

```diff
@@ -0,0 +1,22 @@
+const { didTimePassed } = require("./helper")
+
+const getExcludedGuilds = (jobs) => {
+  const excGuilds = []
+  jobs.forEach((job) => {
+    if (!job.jobState) {
+      excGuilds.push(job.guildId)
+    }
+
+    if (
+      job.jobInterval && 
+      !didTimePassed(job.lastJobTime, job.jobInterval)
+      ) {
+      excGuilds.push(job.guildId)
+    }
+  })
+  return excGuilds
+}
```

Comment:
This could be condensed for ES6:
```js
const getExcludedGuilds = (jobs) => {
  return jobs.filter((job) => !job.jobState || (job.jobInterval && !didTimePassed(job.lastJobTime, job.jobInterval)))
}
```

Reply:
Not quite actually, since this function should return only the ids of the guilds, and not whole objects. But I will change the name, because it is not clear enough what it does.

Reply:
Oh, woops - missed that. You can then use:

```js
const getExcludedGuilds = (jobs) => {
  return jobs
    .filter((job) => !job.jobState || (job.jobInterval && !didTimePassed(job.lastJobTime, job.jobInterval)))
    .map((job) => job.guildId)
}
```